### PR TITLE
834 export metadata subfield related works

### PR DESCRIPTION
**Pull request by sharpattack**

### CODE REVIEW

File: app/models/invenio_rdm_record_converter.rb [hunk 1]
```diff
@@ -286,4 +288,32 @@ def version(content)
 
     "v#{version_number}.0.0"
   end
+
+  def related_identifiers(related_url)
```
Comment (Meowcenary) [inline app/models/invenio_rdm_record_converter.rb:292]: This is a good function. Very clear.

File: app/models/invenio_rdm_record_converter.rb [hunk 2]
```diff
@@ -286,4 +288,32 @@ def version(content)
...
+          "scheme": "url",
+          "relation_type": {"id": "isRelatedTo"}
+        }
+      end
+    end
+
+    identifiers.compact
+  end
+
+  def doi_url?(url)
```
Comment (Meowcenary) [inline app/models/invenio_rdm_record_converter.rb:316]: Nice!

**Review by Meowcenary — APPROVED** [review]
Looks good to me. Nice job with this.

File: app/models/invenio_rdm_record_converter.rb (hunk 1 above)
Comment (sharpattack) [reply app/models/invenio_rdm_record_converter.rb:292]: This is your work actually 😄

File: app/models/invenio_rdm_record_converter.rb (hunk 2 above)
Comment (sharpattack) [reply app/models/invenio_rdm_record_converter.rb:316]: Same here, 😎

**Pull request merged**